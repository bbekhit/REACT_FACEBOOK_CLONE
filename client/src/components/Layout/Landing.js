import React from "react";
import img1 from "../../assets/landing-1.svg";
import Typography from "@material-ui/core/Typography";
import Grid from "@material-ui/core/Grid";

const Landing = () => {
  return (
    <Grid
      container
      style={{
        minHeight: "100vh",
        background: "#c6feff",
        marginTop: "-3rem",
        padding: "2rem"
      }}
      justify="space-around"
      alignItems="center"
    >
      <Grid item sm>
        <Typography variant="h3" color="primary">
          Welcome to the, PARTY
        </Typography>
      </Grid>
      <Grid item sm>
        <img
          src={img1}
          alt="landing"
          style={{ height: "100%", width: "100%" }}
        />
      </Grid>
    </Grid>
  );
};

export default Landing;
