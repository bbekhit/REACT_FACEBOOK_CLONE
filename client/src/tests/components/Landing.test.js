import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import Landing from "../../components/Layout/Landing";

test("should render Landing page correctly", () => {
  const wrapper = shallow(<Landing />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
