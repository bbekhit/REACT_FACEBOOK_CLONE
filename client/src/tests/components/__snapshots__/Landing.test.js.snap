// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Landing page correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  container={true}
  justify="space-around"
  style={
    Object {
      "background": "#c6feff",
      "marginTop": "-3rem",
      "minHeight": "100vh",
      "padding": "2rem",
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={true}
  >
    <WithStyles(ForwardRef(Typography))
      color="primary"
      variant="h3"
    >
      Welcome to the, PARTY
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={true}
  >
    <img
      alt="landing"
      src="landing-1.svg"
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
